### PR TITLE
Added BinaryClasification_TitanicSurvivalPrediction to samples

### PR DESCRIPTION
**Pull request by cmendible**
Hi, this is the addapted BinaryClasification sample based on the [titanic.ml](https://github.com/cmendible/dotnetcore.samples/tree/master/titanic.ml) project I created for my post [Simple Machine Learning with .NET Core Sample](https://carlos.mendible.com/2018/06/10/simple-machine-learning-with-dotnet-core-sample/)

Thanks for letting me contribute!

### CODE REVIEW

**Comment by cmendible** [comment]
Hi @OliaG just moved it to the examples folder. Let me know is any further action is needed.

File: datasets/README.md [hunk 1]
```diff
@@ -22,6 +22,12 @@ The datasets are provided under the original terms that Microsoft received such
 
 > Original source: https://en.wikipedia.org/wiki/Iris_flower_data_set#Use_of_the_data_set.
 
+### Titanic
+
+> Redistributing the dataset "titanic-train.csv" and "titanic-test.csv" with attribution:
+
+> Original source: https://github.com/brinkar/real-world-machine-learning/blob/master/data/titanic.csv
```
Comment (OliaG) [inline datasets/README.md]: Could you please update this lines to:

>Redistributing the dataset "taxi-fare-test.csv", "taxi-fare-train.csv" with attribution:

>Data obtained from http://biostat.mc.vanderbilt.edu/DataSets

>Original source: http://biostat.mc.vanderbilt.edu/wiki/pub/Main/DataSets/titanic.html

File: samples/examples/BinaryClasification_Titanic/BinaryClasification_TitanicSurvivalPrediction/Program.cs [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+        private static async Task Main(string[] args)
+        {
+            // STEP 1: Create a model
+            var model = await TrainAsync();
+
+            // STEP2: Test accuracy
+            Evaluate(model);
+
+            // STEP 3: Make a prediction
+            var prediction = model.Predict(TestTitanicData.Passenger);
```
Comment (OliaG) [inline samples/examples/BinaryClasification_Titanic/BinaryClasification_TitanicSurvivalPrediction/Program.cs]: I have a suggestion. How about we remove one passenger from the test dataset, get his/her parameters into the TestTitanicData.Passenger and do a prediction on it. That way our model would not be exposed to the data before which is more clear experiment.

File: samples/examples/BinaryClasification_Titanic/BinaryClasification_TitanicSurvivalPrediction/Program.cs [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+            Console.WriteLine($"Did this passenger survive? {(prediction.Survived ? "Yes" : "No")}");
```
Comment (OliaG) [inline samples/examples/BinaryClasification_Titanic/BinaryClasification_TitanicSurvivalPrediction/Program.cs]: Would also suggest to word it "Did this passenger survive? Predicted: ... Actual : ..."
And just to be on the optimistic note maybe we can pick a one that survived? :) But that's only my preference.

**Comment by OliaG** [comment]
@cmendible This looks amazing, thank you so much for your contribution. I have just a few minor comments and we'll check this PR in.

File: samples/examples/BinaryClasification_Titanic/BinaryClasification_TitanicSurvivalPrediction/Program.cs (hunk 2 above)
Comment (cmendible) [reply samples/examples/BinaryClasification_Titanic/BinaryClasification_TitanicSurvivalPrediction/Program.cs]: FIxed in: 027b4be

File: samples/examples/BinaryClasification_Titanic/BinaryClasification_TitanicSurvivalPrediction/Program.cs (hunk 3 above)
Comment (cmendible) [reply samples/examples/BinaryClasification_Titanic/BinaryClasification_TitanicSurvivalPrediction/Program.cs]: Fixed in bebf34c

File: datasets/README.md (hunk 1 above)
Comment (cmendible) [reply datasets/README.md]: Fixed in 027b4be

File: samples/examples/BinaryClasification_Titanic/BinaryClasification_TitanicSurvivalPrediction/Program.cs (hunk 3 above)
Comment (cmendible) [reply samples/examples/BinaryClasification_Titanic/BinaryClasification_TitanicSurvivalPrediction/Program.cs]: Thanks for the comments! It looks better now....

**Pull request merged**